File: Java/Bubble_Sort.java
public class Bubble_Sort{

    public static void printArray(int[] arr){
        for(int i=0;i<arr.length;i++){
            System.out.print(arr[i]+" ");
        }
    }
    
    public static void main(String[] args) {
        int arr[]={7,8,3,2,1};
        //bubble sort
        for(int i=0;i<arr.length-1;i++){
            for(int j=0;j<arr.length-1-i;j++){
                if (arr[j]>arr[j+1]){
                    int temp=arr[j];
                    arr[j]=arr[j+1];
                    arr[j+1]=temp;

                }
            }
        }

        printArray(arr);

    }
}

File: Python/bubblesort.py
def bubble_sort(arr):
    n = len(arr)
    for i in range(n):
        swapped = False
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
                swapped = True
        if not swapped:
            break

# list to be sorted, you can change it according to you
arr = [39, 12, 18, 85, 72, 10, 2, 18]
print("Unsorted list is:", arr)
bubble_sort(arr)
print("Sorted list is:", arr)
